core/tests/protractor_desktop/userJourneys.js

```diff
@@ -202,12 +202,12 @@ describe('Site language', function() {
       await users.createUser('audioPlayer@example.com', 'audioPlayer');
       await users.login('audioPlayer@example.com');
       await preferencesPage.get();
-      await preferencesPage.expectPreferredAudioLanguageNotToBe('Chinese');
-      await preferencesPage.selectPreferredAudioLanguage('Chinese');
+      await preferencesPage.expectPreferredAudioLanguageNotToBe('中文 (Chinese)');
```

Comment:
You mentioned that non-english characters result in flakiness, however, you left these characters in here. Is there a particular reason for this?

Reply:
In contributor dashboard there are places where we are using dynamic protractor classes which depend on language description like - `protractor-test- translation-{{ language-description }}` but in preference dashboard there were no such cases. So changing the language in preference dashboard was not required. Also at many places in contributor dashboard 'toBe' is used instead of 'toEqual' which also makes it difficult for non english characters.

Reply:
Makes sense. Thanks for clarifying!